external/Kconfig

```diff
@@ -1,33 +1,34 @@
 
-config DM
-	bool "DM"
-	default n
-	depends on NET
-	---help---
-		enable the DM functionality
-if DM
 config LWM2M_WAKAAMA
```

Comment:
If you refer to wakaama/examples/client/object_connectivity_moni.c, in the function prc_set_value, we use the dm APIs to get RSSI, as an example. The proposed plan was to have similar dm API calls wherever there is a read request from wakaama server. In case wakaama and DM are separated, then there is a larger question of which API would be used in wakaama to access connectivity-based values and functions. Hence my comment above. I understand your problem statement, and would suggest taking this observation into context as well.